recipe/meta.yaml

```diff
@@ -1,43 +1,48 @@
 {% set name = "jmespath" %}
-{% set version = "0.10.0" %}
+{% set version = "1.0.1" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9
+  sha256: 90261b206d6defd58fdd5e85f478bf633a2901798906be2ad389150c5c60edbe
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: true  # [py<37]
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv"
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
```

Reply:
That's totally valid YAML, so I won't do it.